examples/solids/index.rst

```diff
@@ -0,0 +1,459 @@
+.. _example-petsc-elasticity:
+
+Solid mechanics elasticity mini-app
+========================================
+
+This example is located in the subdirectory :file:`examples/solids`.
+It solves the steady-state static momentum balance equations using unstructured high-order finite/spectral element spatial discretizations.
+As with the :ref:`example-petsc-navier-stokes` case, the solid mechanics elasticity example has been developed using PETSc, so that the pointwise physics (defined at quadrature points) is separated from the parallelization and meshing concerns.
+
+In this mini-app, we consider three formulations used in solid mechanics applications: linear elasticity, Neo-Hookean hyperelasticity at small strain, and Neo-Hookean hyperelasticity at finite strain.
+We provide the strong and weak forms of static balance of linear momentum in the small strain and finite strain regimes.
+The stress-strain relationship (constitutive law) for each of the material models is provided.
+Due to the nonlinearity of material models in Neo-Hookean hyperelasticity, the Newton linearization of the material models is provided.
+
+.. note::
+
+   Linear elasticity and small-strain hyperelasticity can both by obtained from the finite-strain hyperelastic formulation by linearization of geometric and constitutive nonlinearities.
+   The effect of these linearizations is sketched in the diagram below, where :math:`\bm \sigma` and :math:`\bm \epsilon` are stress and strain, respectively, in the small strain regime, while :math:`\bm S` and :math:`\bm E` are their finite-strain generalizations (second Piola-Kirchoff tensor and Green-Lagrange strain tensor, respectively) defined in the reference configuration, and :math:`\mathsf C` is a linearized constitutive model.
+
+   .. math::
+      \begin{matrix}
+      \text{Finite Strain Hyperelastic} & \underset{\bm S = \mathsf C \bm E}{\overset{\text{constitutive}}{\longrightarrow}} & \text{St. Venant-Kirchoff} \\
+      \text{\scriptsize geometric} {\LARGE \ \downarrow\ } \scriptsize{\bm E \to \bm \epsilon} & & \text{\scriptsize geometric} {\LARGE \ \downarrow\ } \scriptsize{\bm E \to \bm \epsilon} \\
+      \text{Small Strain Hyperelastic} & \underset{\bm \sigma = \mathsf C \bm \epsilon}{\overset{\text{constitutive}}{\longrightarrow}} & \text{Linear Elastic} \\
+      \end{matrix}
+
+.. _running-elasticity:
+
+Running the mini-app
+----------------------------------------
+
+.. toctree::
+   :hidden:
+
+   README
+
+.. include:: README.rst
+   :start-after: inclusion-marker-do-not-remove
```

Comment:
This is rendering correctly in HTML but not PDF.

Reply:
If something deosn't render properly in one build or another, and you want to exclude that directive from one specific build, you can use the `.. only:: html` for instance or `.. only:: latexf`, as we've done to exlude the mermaid graph in the API doc that was not rendering correctly in the PDF build